### ISSUE TITLE
[FORMATTING] Handle Polymorphic Table Functions formatting in Trino

### ISSUE THREAD

**Issue by dedep** [sql-formatter-org/sql-formatter#509]
**Input data**

Which SQL and options did you provide as input?

```sql
SELECT
  *
FROM
  TABLE (
    postgresql.system.query (query => 'SELECT * FROM tiny.orders LIMIT 100')
  );
```

**Expected Output**

```sql
SELECT
  *
FROM
  TABLE (
    postgresql.system.query (query => 'SELECT * FROM tiny.orders LIMIT 100')
  );
```

**Actual Output**

```sql
SELECT
  *
FROM
  TABLE (
    postgresql.system.query (query = > 'SELECT * FROM tiny.orders LIMIT 100')
  );
```

**Usage**

- How are you calling / using the library?
`format(sql, { language: "trino", linesBetweenQueries: 1, tabWidth: 2 });`
- What SQL language(s) does this apply to?
Trino
- Which SQL Formatter version are you using?
11.0.2

More about PTF - https://trino.io/docs/current/functions/table.html

**Comment by nene** [comment]
Thanks for reporting.

Should be a simple fix. Simply need to add support of the `=>` operator to Trino.

**Comment by nene** [comment]
This is now fixed in 12.0.0-beta.3 release.

**Comment by dedep** [comment]
Thank you
